doc/sphinx/source/libCEEDapi.md

```diff
@@ -307,16 +307,22 @@ specific devices (e.g. GPUs) or specific polynomial orders. A simple reference
 backend implementation is provided in the file
 [ceed-ref.c](https://github.com/CEED/libCEED/blob/main/backends/ref/ceed-ref.c).
 
+(fig-operator-schematic)=
+
+:::{figure} ../../img/libceed_schematic.png
+Flow of data through vector types inside libCEED Operators
```

Comment:
Hi @nbeams thank you so much for adding this and renaming all the G's with \mathcal{E}. Would you consider adding a bit of descriptive text of this image, perhaps above in section "Terminology and Notation" (L109), where we introduce our "Vector representation/storage categories"

Reply:
Hmm, do you think the schematic would fit better in a different location?  I'm not sure if the descriptive text should be so far away from the figure itself...?